suite-native/module-home/src/screens/HomeScreen/components/PortfolioContent.tsx

```diff
@@ -52,7 +52,9 @@ export const PortfolioContent = forwardRef<PortfolioContentRef>((_props, ref) =>
     return (
         <VStack spacing="large" marginTop="small">
             <PortfolioGraph ref={graphRef} />
-            <Assets />
+            <Box marginHorizontal="medium">
```

Comment:
nit: You could remove `marginHorizontal` on this box and the one below and adjust spacing of VStack wrapper

Reply:
I will leave this as it is for now, it won't be that straighforward